sdk/ios/Configsum/Configsum/Configsum.xcodeproj/xcuserdata/alexsavu.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Are these personal files supposed to be committed?

Reply:
I'm still not sure that alexsavu.xcuserdatad is a good thing to commit :)

Reply:
it's in the git  ignore

Reply:
@alexsavu Yes, but they're still in the PR. Can you do a `git rm` on them?